src/main/resources/templates/recipe/detail.html

```diff
@@ -50,6 +50,11 @@
 			<th>Recipe Image:</th>
 			<td><img class="img-fluid img-thumbnail" th:src="${recipe.image}"/></td>
 		</tr>
+
+		<tr>
+			<th>Created By:</th>
+			<td th:text="${users.username}">Default Text</td>
```

Comment:
I think you're on the right track here, but instead of showing it as users.username you're going to want to specifically pull the user attached to the recipe. It should look something like `${recipe.user.username}`